test/test_proxy_tensor.py

```diff
@@ -1946,6 +1946,7 @@ def f(t):
     xfail('angle', ''),
     xfail('argmax', ''),
     xfail('argmin', ''),
+    xfail('batch_norm_with_update', ''),
```

Comment:
Sounds bad?

Reply:
This is actually just inherited from `_native_batch_norm_legit` (L1944)